lib/tether/toolbox.go

```diff
@@ -108,8 +108,19 @@ func (t *Toolbox) InContainer() *Toolbox {
 	return t
 }
 
+func (t *Toolbox) session() *SessionConfig {
```

Comment:
Copying the config/session object is non-trivial. I am going to stash the session pointer instead, and lock the session object instead. Also will put a lock around when the pointer is updated in Reload().